simple/php/rename-mehod_before.php

```diff
@@ -0,0 +1,13 @@
+class Customer {
+  $name = "name";
+  $surname = "surname";
+
+  /**
+   * Method returns customer's surname
+   */
+  function getsnm() {
+    return this->$surname;
```

Comment:
Correct form: $this->surname

Make sure that rest of PHP examples doesn't have this issue, please.